### PR TITLE
Added a coolDownAnnounce function

### PR DESCRIPTION
**Pull request by CreativeCodeCat**
I have added a toggleable message for when the adventure is able to be ran again!

toggle for the pannel
command thats ran to toggle the cooldown announcement
output after the cooldown has ended

**You may delete this note from your Pull Request if you so desire. It is acceptable if you leave it in place.**

### CODE REVIEW

File: javascript-source/games/adventureSystem.js [hunk 1]
```diff
@@ -377,6 +379,11 @@
         clearCurrentAdventure();
         temp = "";
         $.coolDown.set('adventure', false, coolDown, false);
+        if (coolDownAnnounce) {
+            setTimeout(function() {
+                $.say($.lang.get('adventuresystem.reset', $.pointNameMultiple));
+            }, coolDown);
```
Comment (BrandenB) [inline javascript-source/games/adventureSystem.js]: Add `*1000` here or the cooldown will be in milliseconds.

**Review by BrandenB — CHANGES_REQUESTED** [review]

**Comment by CreativeCodeCat** [comment]
@IllusionaryOne Any chance you can have a look and see if this is how it should be :)

Confermation that the command does toggle

**Comment by IllusionaryOne** [comment]
This looks good to me.  If @ScaniaTV is good to sign off, and the changes he requested look good to him, then we can merge.

As a side note, we are implementing two developer reviews and two developer sign offs now on our end.

Cheers!

**Comment by BrandenB** [comment]
Looks good here too, merging! Thanks!

**Review by BrandenB — APPROVED** [review]

**Pull request merged**